client/src/components/CreateAccount.js

```diff
@@ -1,42 +1,95 @@
 import React,{ useState, useEffect } from 'react'
+import { useNavigate } from 'react-router-dom'
+
+import API from '../API'
 
 export const CreateAccount = () => {
     useEffect(() => {
         document.body.className = 'bodyLogin';
         return () => { document.body.className = ''; }
-      });
-
-  return (
-    <div className = 'containerLogin'>
-    <div className = 'CreateAccount'>
-        <div class='createaccTitle'>Create account</div>
-        <form className = 'add-form'>
-            <div className= 'form-control'>
-                <label>Your Name*</label>
-                <input required type='text' placeholder = 'Enter Full Name'/>
-            </div>
-            <div className= 'form-control'>
-                <label>UCSD Email*</label>
-                <input required type='text' placeholder = 'Enter Email' />
-            </div>
-            <div className= 'form-control'>
-                <label>Password*</label>
-                <input required type='text' placeholder = 'Password' />
-            </div>
-            <div className= 'form-control'>
-                <label>Re-enter Password*</label>
-                <input required type='text' placeholder = 'Re-type password' />
-            </div>
-        </form>
-        <button type="button" className = 'createbtn'>Create your account</button>
-        <div className = 'backtoLogin'>
-            <div style={{marginBottom: "10px"}}>* Required Fields</div>
-            <div>Already have an account?</div> 
-            <a href='/'> Back to Login</a>
-        </div> 
-    </div>
-    </div>
-  )
+    });
+
+    const navigate = useNavigate();
+    
+    const createUser = async (e) => {
+        e.preventDefault();
+        const payload = {
+            user: {
+                firstName: e.target.firstname.value,
+                lastName: e.target.lastname.value,
+                email: e.target.email.value,
+                password: e.target.password.value,
+            }
+        }
+       const response = await API.createUser(payload);
+       if(response.status === 200){
+           alert('Created successfully');
+           navigate('/');
+       }
+       else{
+           alert('Email already in use');
+       }
+    }
+
+    /* Password validation */
+    const [disabled, setDisabled] = useState(true);
+
+    const [passwords,setPasswords] = useState({password: '', rePassword: ''})
+
+    const checkPass = (e) => {
+        if(e.target.name === 'password'){
+            passwords.password = e.target.value
+        }
+        else if(e.target.name === 'rePassword'){
+            passwords.rePassword = e.target.value
+        }
+        if(passwords.password !== '' && passwords.rePassword !== '' && passwords.password === passwords.rePassword){
+           setDisabled(false)
+        }
+        else{
+            setDisabled(true)
+        }
+        setPasswords({...passwords})
+        console.log(passwords)
+        console.log(disabled)
+    }
```

Comment:
Note that while it's fine to keep console.logs in this project since we aren't deploying it, we would normally remove log statements if we were to push this kind of application into production